network/edge-services/how-to/terminate-edge-services.mdx

```diff
@@ -0,0 +1,39 @@
+---
+meta:
+  title: How to terminate an Edge Services subscription
+  description: Find out how to terminate your Scaleway Edge Services subscription whenever you want, at the click of a button.
+content:
+  h1: How to terminate an Edge Services subscription
+  paragraph: Find out how to terminate your Scaleway Edge Services subscription whenever you want, at the click of a button.ow to choose the best plan for your needs and change your plan at the click of a button.
+dates:
+  validation: 2024-10-15
+  posted: 2024-10-15
+tags: object-storage edge-services subscription-plan subscribe billing pricing terminate cancel
+categories:
+  - network
+---
+
+Edge Services is an additional feature for Scaleway Load Balancers and Object Storage buckets. It lets you  benefit from a powerful caching service to reduce the load on your origin servers and streamline delivery. On top of this, you get a customizable Edge Services endpoint and can set it to a subdomain of your choice, securing it with an SSL/TLS certificate. To use Edge Services, you must subscribe to a monthly plan.
+
+## How to terminate your Edge Services subscription
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- An active [Edge Services subscription](/network/edge-services/subscribe-edge-services/)
+- [Deleted all your Edge Services pipelines](/network/edge-services/delete-pipeline/)
+
+You can terminate your Edge Services subscription at any time. You will be charged pro-rata for the month you cancel, based on how much of that month your subscription was active.
```

Comment:
Maybe state that the user should have deleted all its Pipelines before, to be able to unsubscribe